scripts/generateServiceAccounts.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/rand"
+	"os"
+	"time"
+)
+
+var (
+	serviceIDChars = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789")
+	r              *rand.Rand
+	services       = []string{"babbage",
+		"florence",
+		"dp-frontend-router",
+		"dp-frontend-filter-dataset-controller",
+		"dp-frontend-renderer",
+		"dp-frontend-dataset-controller",
+		"dp-import-tracker",
+		"dp-dimension-extractor",
+		"dp-dimension-importer",
+		"dp-observation-extractor",
+		"dp-observation-importer",
+		"dp-import-api",
+		"dp-code-list-api",
+		"dp-dataset-api",
+		"dp-filter-api",
+		"dp-recipe-api",
+		"dp-code-list-api",
+		"dp-dataset-exporter",
+		"dp-hierarchy-api",
+		"dp-hierarchy-builder",
+		"dp-dataset-exporter",
+		"dp-dataset-exporter-xlsx",
+		"dp-search-builder",
+		"dp-search-api",
+		"dp-api-router",
+		"dp-download-service",
+		"dp-frontend-geography-controller",
+		"dp-identity-api"}
+)
+
+func main() {
+
```

Comment:
Possibly overkill since we know none exist at the momement _but_ it might be worth getting a map of the services that already exist before starting. This would make the script more robust for future uses.

Basically walk the dir, read each json file and put the service name in a map. Then when creating the services check if it already exists before creating it?